test/smoke-limbo/veccopy-ompt-target-cmake/performBuildRun.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+usage() { echo "Usage: $0 [-a <AOMP directory>]" \
+          1>&2; exit 1; }
+
+while getopts ":Ga:" o; do
+    case "${o}" in
+        a)
+            AOMP_DIR=${OPTARG}
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+
+# Set 'PROJECT_NAME' to the parent directory's name
+PROJECT_NAME=$(basename $(pwd))
+
+# If 'AOMP_DIR' was not specified fallback to user's AOMP directory
+if [[ -z ${AOMP_DIR} ]]; then
+  AOMP_DIR="/home/$USER/rocm/aomp"
+fi
+
+if [ ! -d ${AOMP_DIR} ]; then
+  echo "WARNING: AOMP directory '${AOMP_DIR}' does not exist!"
+fi
+
+echo " >>> Configure ..."
+cmake -B build -S . -DAOMP_DIR=${AOMP_DIR}
+
+echo " >>> Clean & Build ..."
+cmake --build build --clean-first --parallel
```

Comment:
Check for return code and error-out or use the `cmd || exit 1`  style to do this. Here and below

Reply:
Thanks! I opted for the latter `cmd || exit 1`.